src/filebrowser/file-browser-dialog.cpp

```diff
@@ -427,6 +431,22 @@ void FileBrowserDialog::createLoadingFailedView()
     layout->addWidget(label);
 }
 
+void FileBrowserDialog::createEmptyView()
+{
+    empty_view_ = new QWidget(this);
+
+    QVBoxLayout *layout = new QVBoxLayout;
+    empty_view_->setLayout(layout);
+
+    QLabel *label = new QLabel;
+    label->setObjectName(kEmptyFolderLabelName);
+    QString label_text = tr("This folder is empty.");
+    label->setText(label_text);
+    label->setAlignment(Qt::AlignCenter);
+
+    layout->addWidget(label);
```

Comment:
Why creating a parent widget + layout + QLabel? I think a QLabel is all we need.